Player.gd

```diff
@@ -227,7 +226,7 @@ func on_kill(reward):
 		return
 	xp += reward
 	did_persisted_props_change = true
-	
+	#get_parent().objective_complete("Kill the Enemy")
```

Comment:
Debug output?

Reply:
Those can be removed, they were just for testing that objective completion is working